app/mailers/instructor_alert_mailer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class InstructorAlertMailer < ApplicationMailer
+  def email(alert)
+    # return unless Features.email?
```

Comment:
This should not be commented out.

Reply:
To get around the problem of being able to preview the mail, even with `Features.email?` off, see how some of the other mailers are structured, for example, WikiEmailMailer. The idea is to use a class method to call it from other application code, and have the `email` method only called either from the preview or from that class method. That way, we still get the guard clause based on whether email is enabled (and whatever else needs it), and we can still get a working preview.

Reply:
yes ok i will check this